### DISCUSSION TITLE
WYSIWYG Interface CSS for External Apps

### DISCUSSION THREAD

**Discussion by HitomiTenshi** [directus/directus#10722] [Archive]
Is there a styles `css` / `scss` file that has some `class` to apply to a parent element that contains the HTML content from a WYSIWYG field?

WYSIWYG stands for "what you see is what you get". It's meaning is correct when used only within directus, but most of the time I want to render the HTML generated by this field in my own app. It would be nice if there were styles that I can apply to make it look just like it looked in the WYSIWYG field in directus. That way the user does actually get what he saw.

**Comment by benhaynes** [comment]
I agree that we should have a `CSS Override` Interface Option on the WYSIWYG. As a stop-gap solution, you might be able to use the **Project Settings > CSS Override** to target the interface and add new styles... though this might not work since that interface injects everything into an iframe.

**Reply by HitomiTenshi** [reply]
@benhaynes This is not exactly what I was asking. I was asking if there is a WYSIWYG css styling file that I can use in my **external** app (e. g. Angular app) which uses the directus API to query **and render** the HTML data from a WYSIWYG field.

Right now I have to inspect the styles from the WYSIWYG interface and re-apply everything to my own **external** app, so the HTML looks exactly the same in my **external** Angular app, as it did in the WYSIWYG field in directus.

**Reply by benhaynes** [reply]
Ahh, interesting... glad you like our styling! Not sure about dynamically pulling this in, but it doesn't really change, so you can use this:

https://github.com/directus/directus/blob/main/app/src/interfaces/input-rich-text-html/get-editor-styles.ts
